tools/generate.py

```diff
@@ -601,6 +612,21 @@ def _gi_build_stub(
 
     # Functions
     for name in sorted(functions):
+        if hasattr(functions[name], "is_deprecated"):
+            deprecated = functions[name].is_deprecated()
+            if deprecated:
+                full_name = _generate_full_name(prefix_name, name)
+                message = (
+                    functions[name].get_attribute("deprecated")
```

Comment:
Maybe add a comment for easier tracking:

```suggestion
                    # Currently not implemented: https://gitlab.gnome.org/GNOME/gobject-introspection/-/issues/80
                    functions[name].get_attribute("deprecated")
```